service/__init__.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016, 2019 John J. Rofrano. All Rights Reserved.
```

Comment:
We don't need to include the professor's name here.

Reply:
cleanup